appointment/upload-documents/uploadDocumentsDataService.js

```diff
@@ -69,6 +69,39 @@ define('uploadDocumentsDataService', [], function($http) {
 				}, (error) => {
 					return null;
 				});
+			},
+			storeConsent: function(reviewConsent, virtualConsent, signature) {
+				return $http({
+					url: '/server/appointment/upload-documents/uploadDocuments.php',
+					method: 'POST',
+					data: 'action=storeConsent' +
+						`&reviewConsent=${reviewConsent}` +
+						`&virtualConsent=${virtualConsent}`+
+						`&signature=${signature}`,
+					headers: {
+						'Content-Type': 'application/x-www-form-urlencoded'
```

Comment:
Do you know what this does?

Reply:
IIRC, it adds a header to the HTTP request that tells the recipient that the values are urlencoded in the URL path.